### DISCUSSION TITLE
Removing a fragment of song title from all tracks

### DISCUSSION THREAD

**Discussion by awajac** [RudeySH/lastfm-bulk-edit#41] [Ideas]
You know when you have from Spotify not onyl album "remastered" but also all tracks are "remastered". Now we can edit album for all tracks or all scrobbless of any track.
Do you think is it possible to remove part of title from all tracks from one album or even ALL tracks. 
Example:
U2 - Joshua Tree
           
Trip Through Your Wires - Remastered 2007
With Or Without You - Remastered 2007

Maybe it can be as additional option in album edit window. Option "remove part of title", I can type "Remastered 2007" and it will be removed from titles and I'll get
Trip Through Your Wires
With Or Without You

**Comment by BigDawgBen** [comment]
This would be a great feature. At present, it has to be done on a track-by-track basis. It would be ideal to be able to bulk edit scrobbles for a track/album/artist and remove additional text (e.g., "Remastered version 2018"). This could be typed into a dialogue box and the edit just removes that text from every loaded scrobble.
